change-beta/@azure-communication-react-3403a61e-01e3-4fc3-a75c-6b80a7e66cf2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
note: for cherry picks into release branch we should be updating the changelog directly in this PR since its already been generated (although then its not strictly a cherry pick 🤔)